encryption/src/main/java/org/apache/solr/encryption/EncryptionRequestHandler.java

```diff
@@ -194,15 +258,22 @@ public void handleRequestBody(SolrQueryRequest req, SolrQueryResponse rsp) throw
     } else if (keyId.equals(NO_KEY_ID)) {
       keyId = null;
     }
+    // Check the defined DirectoryFactory instance.
     EncryptionDirectoryFactory.getFactory(req.getCore());
+
+    if (req.getParams().getBool(DISTRIB, false)) {
```

Comment:
It'd be nice if we could default "distrib" based on whether the request was addressed to the collection or not.  Maybe that metadata is there?

Reply:
I don't see this metadata. The collection name is present in the CloudSolrClient.request call, but not passed to the inner LBSolrClient.request call.

Reply:
`SolrQueryRequest.getHttpSolrCall().getCoreOrColName()` -- but never mind my thought on this.  Since SearchHandler/CloudReplicaSource doesn't toggle its scope based on looking at this, we shouldn't either.  Consistency.